frontend/packages/dev-console/src/components/import/devfile/DevfileImportForm.tsx

```diff
@@ -33,11 +65,31 @@ const DevfileImportForm: React.FC<FormikProps<FormikValues> & DevfileImportFormP
             {devfileParseError}
           </Alert>
         )}
-        <GitSection
-          buildStrategy="Devfile"
-          builderImages={builderImages}
-          defaultSample={{ url: 'https://github.com/redhat-developer/devfile-sample' }}
-        />
+        {formType === 'sample' ? (
+          <FormSection>
+            <InputField
+              type={TextInputTypes.text}
+              name="git.url"
+              label={t('devconsole~Git repo URL')}
+              data-test-id="git-form-input-url"
+              isDisabled
+            />
+            {selectedBuilderImage && (
+              <ImageStreamInfo
+                displayName={selectedBuilderImage.name}
+                tag={selectedBuilderImage.tags[0]}
+              />
```

Comment:
This info should be about selected devfile. The API returning sample devfile can be called here to get all devfiles and then select and show information using the metadata returned by the API.